easybuild/easyconfigs/k/Kent_tools/Kent_tools-20190326-linux.x86_64.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'BinariesTarball'
+
+name = 'Kent_tools'
+version = '20190326'
+versionsuffix = '-linux.x86_64'
+
+homepage = 'http://genome.cse.ucsc.edu/'
+description = """Kent tools: collection of tools used by the UCSC genome browser."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Check the last modified date at http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64
+# Then and pack into tarball with:
+#   mkdir Kent_tools
+#   rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/ Kent_tools
+#   tar cfvz Kent_tools-20190326.tar.gz Kent_tools
+#
+sources = [SOURCE_TAR_GZ]
+checksums = ['ce9da071e4e31abdda5ecff882fa9799ef879a4fc9a7d43a0799cff2de965444']
```

Comment:
Also, including a checksum here is pointless, since you need to self-create the tarball, which will typically result in a different checksum on different systems (despite the same tarball contents)